foodji_api/Api/appsettings.json

```diff
@@ -8,6 +8,7 @@
   "AllowedHosts": "*",
   "Auth": {
     "Firebase": {
+      "GoogleApplicationCredentialsFile": "E:/School/Projet_2/Foodji/foodji_api/secrets/foodji-auth-google-credentials.json",
```

Comment:
Ce path là fonctionne seulement pour toi, est-ce qu'on peut avoir un path relatif?

Reply:
Oops, yep! J'ai push un "fix" - c'est loin d'être élégant, mais sans vraie gestion des secrets (tel qu'on en a convenu au départ), c'est pas particulièrement faisable de toute façon. J'ai laissé un commentaire à cet effet dans le .csproj affecté. J'ajouterai bien un revamp config + secrets dans le backlog, mais ce serait 100% une tâches backlog-to-never-be-done dans le contexte du projet...

Reply:
Et là si je comprends bien, on a un fichier .json nécessaire au build du projet auth qui est dans les secrets, donc pas committed sur github et il est nécessaire au build donc le Actions plante? Si c'est ça, est-ce qu'on a un workaround qui permettrait au moins de rouler les tests de Domain? Je pense à un fichier .json du même nom contenant du blob qu'on pourrait commit juste une fois et après avec le git ignore conserver le fichier d'origine et changer son contenu localement?

C'est vraiment shady comme solution mais ça permettrait notamment au CI de rouler!